website/docs/cloud-docs/api-docs/audit-trails-tokens.mdx

```diff
@@ -0,0 +1,147 @@
+---
+page_title: Audit Trails Tokens - API Docs - HCP Terraform
+description: >-
+  Use the `/authentication-token` endpoint with the `token` query param to manage audit trails API tokens. Generate and delete audit trails tokens using the HTTP API.
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+
+[201]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
+
+[202]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
+
+[204]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204
+
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+
+[412]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412
+
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+
+[429]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429
+
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+
+[JSON API document]: /terraform/cloud-docs/api-docs#json-api-documents
+
+[JSON API error object]: https://jsonapi.org/format/#error-objects
+
+# Audit Trails Token API
+
```

Comment:
Can you add a sentence describing what audit trail tokens are and how you use them?

Reply:
Also are these HCP TF plus only? I assume they are because the [Audit Trails API is](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/audit-trails)?

Reply:
Getting some advice on this point.  I can see a usecase where they shouldn't be gated behind the subscription level.

Reply:
So closing this off.  The behaviour here;

In TFC only;

* If an org is NOT entitled to Audit Trails, a user will be able to view and delete tokens, but not create a new one. This is to accomodate the scenario where a token is created and then the org downgrades their subscription.

* If an org IS entitled, a user can create/view/delete an audit token

Reply:
@rkoron007 ^Should I add this as a note under the `Audit Trails Token API` heading?

Reply:
Just for my own curiosity, in the scenario where an organization has an existing audit trail token, but their organization downgrades and no longer has access to audit trails, what happens if someone tries to use that token? Is there an error that explains the issue if they try to use it? If yes, we should probably document that error so that someone CTRL+Fing for it finds it quickly. 

It feels like this might be _longer_ than a callout's worth of information— so, I'd recommend either adding a short callout and then expanding on that information in a paragraph below, or just adding a whole second paragraph at the top of the page. I like the placement suggestion though, @tstormk! Feel free to tag me if you'd like a second set of eyes on the new stuff you add ✨

Reply:
@glennsarti Could you respond to Rose's question before I edit the docs?

Reply:
Whoops missed this one; 

> , but their organization downgrades and no longer has access to audit trails, what happens if someone tries to use that token? Is there an error that explains the issue if they try to use it?

So... I don't think we need anything extra here. What happens is, the same thing now if someone used an Organization token to access audit trails; They get an empty response. There really isn't any change in behaviour as far as using a token that doesn't have audit logging enabled.

Reply:
Might still be worth adding a sentence that this is the behavior _just in case _ someone isn't familiar with the current implementation. Thanks for the clarification though!

Reply:
Fair enough